model/host/host.go

```diff
@@ -26,6 +26,7 @@ type Host struct {
 	Tag      string        `bson:"tag" json:"tag"`
 	Distro   distro.Distro `bson:"distro" json:"distro"`
 	Provider string        `bson:"host_type" json:"host_type"`
+	IPv6	 string		   `bson:"ipv6_address" json:"ipv6_address"`
```

Comment:
You editor should have a thing to align this on save, probably with `goimports`, but potentially with `gofmt`.